From 4d5e4390e77257a31c3c31dac268dc8cd1f68ebb Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Wed, 26 Feb 2020 17:51:57 +0200
Subject: [PATCH 109/186] rework Simple Day font overlay

	modified:   packages/overlays/FontSimpleDayOverlay/AndroidManifest.xml
	modified:   packages/overlays/FontSimpleDayOverlay/res/values/config.xml
---
 packages/overlays/FontSimpleDayOverlay/AndroidManifest.xml   | 2 +-
 packages/overlays/FontSimpleDayOverlay/res/values/config.xml | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/packages/overlays/FontSimpleDayOverlay/AndroidManifest.xml b/packages/overlays/FontSimpleDayOverlay/AndroidManifest.xml
index 905bcb1c71b..2d7c111ea94 100644
--- a/packages/overlays/FontSimpleDayOverlay/AndroidManifest.xml
+++ b/packages/overlays/FontSimpleDayOverlay/AndroidManifest.xml
@@ -26,6 +26,6 @@
     <application android:label="@string/font_simpleday_overlay" android:hasCode="false">
         <meta-data
             android:name="android.theme.customization.REQUIRED_SYSTEM_FONTS"
-            android:value="simple-day,simple-day-medium" />
+            android:value="simple-day,simple-day-bold" />
     </application>
 </manifest>
diff --git a/packages/overlays/FontSimpleDayOverlay/res/values/config.xml b/packages/overlays/FontSimpleDayOverlay/res/values/config.xml
index ef1853ce3b9..0ad50d70d8c 100644
--- a/packages/overlays/FontSimpleDayOverlay/res/values/config.xml
+++ b/packages/overlays/FontSimpleDayOverlay/res/values/config.xml
@@ -19,9 +19,9 @@
      <!-- Name of a font family to use for body text. -->
     <string name="config_bodyFontFamily" translatable="false">simple-day</string>
     <!-- Name of a font family to use for medium body text. -->
-    <string name="config_bodyFontFamilyMedium" translatable="false">simple-day-medium</string>
+    <string name="config_bodyFontFamilyMedium" translatable="false">simple-day-bold</string>
     <!-- Name of a font family to use for headlines. If empty, falls back to platform default -->
     <string name="config_headlineFontFamily" translatable="false">simple-day</string>
     <!-- Name of the font family used for system surfaces where the font should use medium weight -->
-    <string name="config_headlineFontFamilyMedium" translatable="false">simple-day-medium</string>
+    <string name="config_headlineFontFamilyMedium" translatable="false">simple-day-bold</string>
 </resources>
-- 
2.17.1

